test/libevmcore/CMakeLists.txt

```diff
@@ -1,5 +0,0 @@
-cmake_policy(SET CMP0015 NEW)
-
-aux_source_directory(. SRCS)
-
-add_sources(${SRCS})
```

Comment:
I removed the whole directory because there was no .cpp file left afterwards.
Also note that you moved some libevmcore tests to the libsolidity folder already. I know that I did not draw a clear line between those tests, I will clean that up when we have more dedicated libevmcore tests.